app/models/project.rb

```diff
@@ -56,6 +56,14 @@ def set_last_activity_at
     update_column(:last_activity_at, self.created_at)
   end
 
+  # create default hooks if they are presented
+  after_create :create_default_web_hooks, if: -> { !!gitlab_config_features.auto_web_hooks }
```

Comment:
Can you please move the after_create to the top of the file? That way we can see in one look what callbacks are being called. Also the comment seems to be redundant, you can remove that one